### ISSUE TITLE
Error: ``Could not open input file: /scripts/script.php``

### ISSUE THREAD

**Issue by pcav** [3liz/lizmap-docker-compose#13]
Upon firtst start:
```
git clone https://github.com/3liz/lizmap-docker-compose.git
make start
```
I get an error:
```
lizmap_1  | + su -c 'php /scripts/script.php jcommunity~user:create -v --no-error-if-exists --admin --reset   admin'
lizmap_1  | Could not open input file: /scripts/script.php
```

**Comment by dmarteau** [comment]
This was a regression in lizmap 3.4.1, the problem is described here: https://github.com/3liz/lizmap-docker-compose/issues/9

**Comment by pcav** [comment]
Thanks. However I'm puzzled: the PR have been merged, I checkoout master, so I should already have the fix. What am I missing?

**Comment by dmarteau** [comment]
You may have to pull-update the docker images.

**Comment by pcav** [comment]
`git pull` says it's already up to date. should I checkout a different origin?
Thanks.

**Comment by dmarteau** [comment]
> `git pull` says it's already up to date. should I checkout a different origin?
> Thanks.

It is not about `git pull` but `docker pull  3liz/lizmap-web-client:<version>`

**Comment by pcav** [comment]
`docker pull 3liz/lizmap-web-client:3.4.0` and .1 give the same error as above.

**Comment by dmarteau** [comment]
@nboisteault @laurentj could you confirm that the fix for #9 has been released with the 3liz docker images ?

**Comment by nboisteault** [comment]
@dmarteau @pcav Fix will land with coming LWC 3.4.2. To use LWC master as a docker image you have to build it : https://github.com/3liz/lizmap-web-client/blob/master/docker/CONTRIBUTING.md

You can also start LWC 3.4.0 with `make start LIZMAP_VERSION_TAG=3.4.0` if you don't need master version.

**Comment by pcav** [comment]
I confirm, it works with the last command.
